cms/djangoapps/contentstore/views/course.py

```diff
@@ -738,8 +743,17 @@ def _create_new_course(request, org, number, run, fields):
     Returns the URL for the course overview page.
     Raises DuplicateCourseError if the course already exists
     """
+    org_data = get_organization_by_short_name(org)
+    if not org_data and organizations_enabled():
+        return JsonResponse(
+            {'error': _('You must link this course to an organization in order to continue. '
+                        'Organization you selected does not exist in the system, '
+                        'you will need to add it to the system')},
```

Comment:
Actually I suggest an autosuggest widget instead of dropwdown and as a result of it we have to show different message. See in the comments of story.